autopush/tests/test_endpoint.py

```diff
@@ -1401,6 +1401,8 @@ def test_base_tags(self):
                                 host='example.com:8080')
         tags = self.reg.base_tags()
         eq_(tags, ['user_agent:test', 'host:example.com:8080'])
+        tags = self.reg.base_tags()
+        eq_(tags, ['user_agent:test', 'host:example.com:8080'])
```

Comment:
Hrm, I'd be interested in knowing why it was failing the test. Add a comment for now if the test must have it to pass.